task-10/js/app.js

```diff
@@ -0,0 +1,105 @@
+// Section 1
+
+console.log(1 + 2);
+console.log("apple" + "orange");
+console.log(1 + 2 + "apple");
+console.log("apple" + 1 + 2);
+console.log(1 + true);
+console.log(0 == false);
+console.log(1 === true);
+console.log(2 == "2");
+
+// Section 2
+
+// 1
+
+cricketPlayers = ["Ashok", "Harish", "Dinesh", "Maamathi", "Kavya", "Ranjith", "Charan", "Hareesh", "Mathi", "Thendral", "Pavithra"]
+
+console.log(cricketPlayers);
```

Comment:
It would be better to have consistent way either to have semicolon or not for the entire app, but only stick to one